.eslintrc.js

```diff
@@ -1,93 +1,120 @@
 module.exports = {
-    parser: "@typescript-eslint/parser",
-    parserOptions: {
-      ecmaVersion: 2018,
-      sourceType: "module",
-    },
-    plugins: ["@typescript-eslint", "json", "react", "react-hooks"],
-    env: {
-      browser: true,
-      es6: true,
-      jest: true
-    },
-    settings: {
-      react: {
-        pragma: "React",
-        version: "detect"
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaVersion: 2018,
+    sourceType: "module",
+  },
+  plugins: ["@typescript-eslint", "json", "react", "react-hooks"],
+  env: {
+    browser: true,
+    es6: true
+  },
+  settings: {
+    react: {
+      pragma: "React",
+      version: "detect"
+    }
+  },
+  ignorePatterns: [
+    "dist/", "node_modules/"
+  ],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:eslint-comments/recommended",
+    "plugin:json/recommended",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended"
+  ],
+  rules: {
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/no-confusing-non-null-assertion": "error",
+    "@typescript-eslint/no-empty-interface": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-require-imports": "error",
+    "@typescript-eslint/no-shadow": ["error", { builtinGlobals: false, hoist: "all", allow: [] }],
+    "@typescript-eslint/no-unused-vars": ["warn", { args: "none", ignoreRestSiblings: true }],
+    "@typescript-eslint/prefer-optional-chain": "warn",
+    "@typescript-eslint/no-var-requires": "off",
+    "curly": ["error", "multi-line", "consistent"],
+    "dot-notation": "error",
+    "eol-last": "warn",
+    "eqeqeq": ["error", "smart"],
+    "eslint-comments/no-unused-disable": "warn",
+    "no-debugger": "off",
+    "no-duplicate-imports": "error",
+    "no-sequences": "error",
+    "no-shadow": "off", // superceded by @typescript-eslint/no-shadow
+    "no-tabs": "error",
+    "no-unneeded-ternary": "error",
+    "no-unused-expressions": ["error", { allowShortCircuit: true }],
+    "no-unused-vars": "off",  // superceded by @typescript-eslint/no-unused-vars
+    "no-useless-call": "error",
+    "no-useless-concat": "error",
+    "no-useless-rename": "error",
+    "no-useless-return": "error",
+    "no-var": "error",
+    "no-whitespace-before-property": "error",
+    "object-shorthand": "error",
+    "prefer-const": "error",
+    "prefer-object-spread": "error",
+    "prefer-regex-literals": "error",
+    "prefer-rest-params": "error",
+    "prefer-spread": "error",
+    "quotes": ["error", "double", { allowTemplateLiterals: true, avoidEscape: true }],
+    "jsx-quotes": ["error", "prefer-double"],
+    "object-curly-spacing": ["error", "always"],
+    "array-bracket-spacing": ["error", "never"],
+    "radix": "error",
+    "react/jsx-curly-spacing": ["error", { "when": "never", "children": { "when": "always" } }],
```

Comment:
```suggestion
    "react/jsx-curly-spacing": ["error", { "when": "never", "children": true }],
```
After playing around with this for a bit, I think this setting is more consistent with existing CC practice. See below for an example.